### ISSUE TITLE
[FEAT] Access files for reading or editing in a sibling webdav container

### ISSUE THREAD

**Issue by mod0** [linuxserver/docker-libreoffice#11]
### Is this a new feature request?

- [X] I have searched the existing issues

### Wanted change

I was wondering if it is possible to access files for editing in a sibling webdav container that is connected to the same custom bridge network?

### Reason for change

This will allow for instance to run a setup where in one container we could be running libreoffice and in another nextcloud. Subsequently, we can edit the files in the nextcloud container via webdav from libreoffice provided they are connected to the same custom bridge.

### Proposed code change

Not yet.

**Comment by thelamer** [comment]
why not just mount the same data into the libreoffice container and edit them all you want in libreoffice, when you click save they will be updated across any containers that have that same bind mount of data no?

**Comment by mod0** [comment]
Two reasons I can think of given that access is through webdav:

1. ACL enforcement
2. Locking/Versioning

**Comment by thelamer** [comment]
I have never used webdav, but docs say you are just opening a file from a URL in libreoffice. Why would this not be possible?

**Comment by mod0** [comment]
Apologies for the delay. This is what I see if I try remote location:

![image](https://github.com/linuxserver/docker-libreoffice/assets/5711446/f4eedeb7-6948-4e54-b413-378a43a7573b)

These are the settings of the connection:

![image](https://github.com/linuxserver/docker-libreoffice/assets/5711446/04ae8acf-962d-43ca-930a-fdb05d5aff0d)

Both the containers are connected to the same custom bridge network on the same machine. My nextcloud instance is named nextcloud.


My nextcloud instance says I need to use this address:

![image](https://github.com/linuxserver/docker-libreoffice/assets/5711446/be91e67a-e122-48f9-9a67-3c175e79d43f)

**Comment by mod0** [comment]
I was wondering whether, because libreoffice is running on top of https://www.kasmweb.com/kasmvnc, if something needs to be done special for networking to work, atleast locally.

**Comment by thelamer** [comment]
Does this work on a normal install of LibreOffice? Their docs reference a process where you need to use the file dialouges to accept a cert and possibly enter auth information: 
https://help.libreoffice.org/latest/en-US/text/shared/guide/digitalsign_receive.html

**Comment by mod0** [comment]
Thank you, you brought up a great point that I overlooked. I tried connecting from my local libreoffice, but to no avail. I was able to connect from cyberduck and nextcloud desktop client. So the connection string is all good.

I am at a point where I think the effort required to resolve this feature request effort is not worth in the long-term given there are other ways of "mounting" the webdav directory locally on each major OS.
